src/Debouncer/Basic.elm

```diff
@@ -431,12 +431,12 @@ so far?
 
 You can use several pre-built accumulators:
 
-    - `lastInput` (the default)
-    - `firstInput`
-    - `allInputs`
-    - `addInputs`
-    - `appendInputToOutput`
-    - `appendOutputToInput`
+  - `lastInput` (the default)
```

Comment:
These blocks need to be indented by 2 spaces to render as list items (as opposed to code blocks, which need to be indented by 4 space).

![screenshot](https://user-images.githubusercontent.com/2716069/49167400-94333400-f335-11e8-92ec-980fa248711f.png)